app/models/app.model.js

```diff
@@ -103,4 +103,30 @@ exports.insertCommentByArticleId = (article_id, username, body) => {
             })
         }
     })
+}
+
+exports.updateArticleById = (article_id, {inc_votes: votes}) => {
+    if(typeof votes !== "number"){
```

Comment:
PSQL should throw an error if this is not a number, so see if you can refactor to use your error handling middleware to catch this instead 👍